src/EX_browser.c

```diff
@@ -3462,7 +3463,7 @@ static info_filter_t* SB_InfoFilter_Parse(int* count)
 
 		if (split != NULL && split[1] && split[1] != '*') {
 			// no regex is fine
-			filter->regex = pcre_compile(split + 1, PCRE_CASELESS, &error, &error_offset, NULL);
+			filter->regex = pcre2_compile((PCRE2_SPTR)(split + 1), PCRE2_ZERO_TERMINATED, PCRE2_CASELESS, &error, &error_offset, NULL);
 			if (filter->regex == NULL) {
 				Con_Printf("Invalid rule definition: %s\n", error);
```

Comment:
You now pass error as an integer Con_Printf's %s when a pointer is expected. This will cause printf to try to access a string at the address of the error code, which is probably an integer at the range of PCRE_COMPILE_BASE+smallnumber, which will cause the application to crash. See gist from yesterday for how to fix the error handling:

https://gist.github.com/dsvensson/8aa1202b8439718a53b04dcd82c24fec

There are a couple of more cases like this that needs fixing.

Reply:
I tried to just change them from %s to %d for now. If you find more than this one please report them.